app/routes/admin/messages/list.js

```diff
@@ -1,7 +1,32 @@
 import Route from '@ember/routing/route';
+import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
 
-export default Route.extend({
-  model() {
-    return this.modelFor('admin.messages');
+export default class extends Route.extend(EmberTableRouteMixin) {
+
+  async model(params) {
+    return this.asArray(this.modelFor('admin.messages'));
   }
-});
+
+}
+// }
+// import Route from '@ember/routing/route';
+// import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
+
+// export default class extends Route.extend(EmberTableRouteMixin) {
+//   titleToken() {
+//     return this.l10n.t('Mail Logs');
+//   }
+
+//   async model(params) {
+//     const searchField = 'recipient';
+//     let filterOptions = this.applySearchFilters([], params, searchField);
+
+//     let queryString = {
+//       filter         : filterOptions,
+//       'page[size]'   : params.per_page || 100,
+//       'page[number]' : params.page || 1
+//     };
+//     queryString = this.applySortFilters(queryString, params);
+//     return this.asArray(this.store.query('mail', queryString));
```

Comment:
won't this be necessary for the tables?

Reply:
I will apply the filters after the table is properly rendered.
Did you find anything which is hindering with the Notification messages ?